dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -380,6 +379,11 @@
       <ThemeResource>true</ThemeResource>
       <Link>Themes\%(Filename)%(Extension)</Link>
     </Page>
+    <Page Include="$(OutDir)rs1_compact_themeresources.xaml">
```

Comment:
I know we'd had conversations about the potential impact on compile time of expanding the combinations of resource dictionaries due to Compact/Preview. But I'm not sufficiently familiar with how this is all set up or what the implications are of your changes. Can @kmahone or @llongley take a look?

Reply:
I have another PR #474 is ongoing, timing is not a issue.

Reply:
Reactivating this conversation until that PR is resolved.

Reply:
@chrisglein  Please review Microsoft.UI.Xaml.vcxproj again.
#474 is merged and GenerateCompactThemeResourceFile is 600ms in my machine.